### DISCUSSION TITLE
find vs search

### DISCUSSION THREAD

**Discussion by gengdz** [LazyVim/LazyVim#3601] [Q&A]
When to use **find** and when to use **search**?

Some examples：
```lua
{ "<leader>ff", LazyVim.pick("auto"), desc = "Find Files (Root Dir)" },
```
Why `<leader>ff` and not `<leader>sf`? 

```lua
 { "<leader>sg", LazyVim.pick("live_grep"), desc = "Grep (Root Dir)" },
```
Why `<leader>sg` and not `<leader>sg`? 

I think it followed a certain principle, what is it?

**Comment by folke** [comment]
Finding files vs search in files
